keel-core/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ResourceRepository.kt

```diff
@@ -170,7 +170,7 @@ interface ResourceRepository : PeriodicallyCheckedRepository<ResourceHeader> {
       } else {
         null
       },
-      locations = if (spec is Locatable) {
+      locations = if (spec is Locatable<*>) {
```

Comment:
Why do you have to specify `<*>` here? (aka what did I miss)

Reply:
I added a generic type to `Locatable` so that I could make the AZ resolver require a spec that is `Locatable<SubnetAwareRegionSpec>`. It doesn't make sense to try to resolve AZs on things that don't have AZs!